pkg/kubelet/kubelet.go

```diff
@@ -2933,29 +2948,14 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 				}
 			}
 		}
-		allocatedPod = pod
```

Comment:
This line should remain in order to pass the new allocation to `kl.isPodResizeInProgress()`.

Reply:
thanks for the catch! fixed